src/main/webapp/WEB-INF/app/views/modals/organization/organizationConfirmRestoreDefaultsModal.html

```diff
@@ -0,0 +1,17 @@
+<div class="modal-header {{attr.modalHeaderClass}}">
+  <button type="button" class="close" ng-click="cancelRestoreOrganizationDefaults()" aria-label="Close">
+    <span class="modal-close" aria-hidden="true">&times;</span>
+  </button>
+  <h3 class="modal-title">Resore Defaults</h3>
+</div>
+
+<form novalidate>
+  <div class="modal-body">
+    <h4>Really restore defaults for "{{ getSelectedOrganization().name }}"?</h4>
+    <p>This will result in the loss of all organizational customizations.</p> 
```

Comment:
We probably want to indicate that all children organizations will have their workflow restored to default as well.